docs/administering/for-work.md

```diff
@@ -9,6 +9,68 @@ Work, and it gives us time to finish testing the payments system.
+organization-related features via the `/admin/organization` settings area.
+
+To apply settings to all users within an organization, you must configure at least one login provider so
+that it knows which people are part of the organization. Users within the organization will automatically
+be able to log in, install apps, and create grains. 
+
+To enable this feature:
+
+- Log in to your server as an administrator.
+
+- Click your name at the top right, then **Admin panel**; this should take you to `/admin`.
+
+- Under the "Configuration" header, click "Organization settings"
+
+- Enable and configure your organization membership on a per-login-provider basis.
+
+Login providers have different settings that are used to define your organization. A user is
+considered a member of your organization if the settings for **at least one** login provider
+declare the user to be a member. You can enable/disable this on a per-login-provider basis.
+
+- **Google authentication.** All users who use a particular Google Apps domain of your choosing can
+  receive user status in Sandstorm. When you enable the use of Google Apps to define the boundary
+  of your organization, you must specify which Google Apps domain represents your organization.
+
+- **LDAP authentication.** All users who log in via LDAP can automatically receive user status in
+  Sandstorm. When you use LDAP to define the boundary of your organization, all users who log in via
+  LDAP are considered members of the organization.
+
+- **SAML authentication.** All users who log in via LDAP can automatically receive user status in
+  Sandstorm. When you use SAML to define the boundary of your organization, all users who log in via
+  SAML are considered members of the organization.
+
+- **Passwordless email login.** All users who use a particular email address domain name
+  (e.g. @example.com) can receive user status in Sandstorm. To enable this, you must specify which
+  Internet domain name represents your organization.
+
+This feature is important because, by default, when a user signs into a Sandstorm server for the
+first time, Sandstorm creates a _Visitor_ account for them. Visitors can use grains that have been
+shared with them but cannot create grains of their own.
+
+### Additional organization settings
+
+Once you have defined your organization, you can optionally configure system-wide rules based on
+organization membership. This can be done from `/admin/organization` within your Sandstorm server.
+
+You can **disallow collaboration with users outside the organization.** If you enable this option,
+grains can only be seen by users logged-in as a member of your organization. This means:
+
+- Unauthenticated (aka anonymous) users cannot view grains, even if they have a sharing link.
+
+- Visitors (logged-in users who have no special permissions on your server) cannot view grains, even
```

Comment:
:+1:

Consider filing an issue which, when closed, would make that Security note obsolete.  I'd really rather have the feature just do the thing the user needs rather than have weird caveats that we document.